app.js

```diff
@@ -32,11 +39,33 @@ io.on('connection', socket => {
 
 	socket.on('drop', dropInfo => {
 		cards[dropInfo.id].row = dropInfo.target
-		fs.writeFileSync('./cards.json', JSON.stringify(cards))
+		fs.writeFileSync(path, JSON.stringify(cards))
 		io.emit('drop', dropInfo)
 	})
 })
 
+// Api Routes
+
+const api = express.Router()
+
+api.post('/task', async (req, res) => {
```

Comment:
Maybe consider making a routes.js file for seperating the routes of the project?

Reply:
Maybe Chippr wants to pick it up and expand it later?